app/src/main/java/org/fossasia/phimpme/gallery/util/ThemeHelper.java

```diff
@@ -26,6 +26,7 @@
 
 import java.lang.reflect.Field;
 import java.lang.reflect.Method;
+
```

Comment:
Un needed one as it's from the java package

Reply:
Two were `reflect` and one was `util`, that's why separated. Sorry.
Reverted :+1: